pytentiostat/LSV-v0.1.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
```

Comment:
please remove all version numbering from the filename.   Git is now handling versions, not the user!  Welcome to the exciting new world of proper version control, rather than filename-versioning!   It is more important than that....user-applied filename versioning will break git because whenever you create a new file, it loses any history that went in to creating it to that point.....so the rule is never change the name of a file.....  

Thing2, standard is that module names are always lower-case, so this file should be named ``lsv.py``